resources/views/components/general/identity.blade.php

```diff
@@ -20,7 +20,7 @@
         throw new Exception('You must provide a model or an address');
     }
 
-    $address = $model ? $model->address() : $address;
+    $address = $model ? $model->address() : $address;    
```

Comment:
```suggestion
    $address = $model ? $model->address() : $address;
```

not sure why none of the workflows picked up on this